### PR TITLE
Replace tabs with spaces

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11713)

### CODE REVIEW

**Comment by MathiasReker** [comment]
@LouiseBonnard No change to wording :)

File: tools/parser_sql/index.php [hunk 1]
```diff
@@ -23,13 +23,13 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-				    	
+
```
Comment (PierreRambaud) [inline tools/parser_sql/index.php]: maybe add a way to remove unneeded whitespaces in your script?

Comment (MathiasReker) [reply tools/parser_sql/index.php]: This is the fix:  `Search (\h*\R){3,}|\h+\R replace \r\n(?1\r\n)`  I will apply it later, I have to go now.

**Comment by MathiasReker** [comment]
@PierreRambaud Check my commits, wdyt?

**Comment by MathiasReker** [comment]
Please don't merge this, before this has been merged: https://github.com/PrestaShop/PrestaShop/pull/11695

**Comment by MathiasReker** [comment]
Please merge this one: https://github.com/PrestaShop/PrestaShop/pull/11781 Then I make a new PR that replace tabs with spaces.

**Pull request closed without merging**